statistics/semantic/src/main/java/com/spotify/heroic/statistics/semantic/SemanticMetadataBackendReporter.java

```diff
@@ -74,17 +75,21 @@ public SemanticMetadataBackendReporter(SemanticMetricRegistry registry) {
             base.tagged("what", "delete-series", "unit", Units.QUERY));
         findKeys = new SemanticFutureReporter(registry,
             base.tagged("what", "find-keys", "unit", Units.QUERY));
-        write =
-            new SemanticFutureReporter(registry, base.tagged("what", "write", "unit", Units.WRITE));
+        write = new SemanticFutureReporter(registry,
+            base.tagged("what", "write", "unit", Units.WRITE));
         backendWrite = new SemanticFutureReporter(registry,
             base.tagged("what", "backend-write", "unit", Units.WRITE));
+
         entries = registry.counter(base.tagged("what", "entries", "unit", Units.COUNT));
 
         writesDroppedByCacheHit = registry.counter(
             base.tagged("what", "writes-dropped-by-cache-hit", "unit", Units.COUNT));
         writesDroppedByDuplicate = registry.counter(
             base.tagged("what", "writes-dropped-by-duplicate", "unit", Units.COUNT));
 
+        // only relevant to es backend.
+        failedShards = registry.counter(
+            base.tagged("what", "failed-es-shards", "unit", Units.COUNT));
```

Comment:
Since this is only for metadata, can we add a tag for that - `component=metadata` or something?

Reply:
@hexedpackets that's already done above on line 64. 

` final MetricId base = MetricId.build().tagged("component", COMPONENT);`